theme

```diff
@@ -41,6 +41,7 @@
   "houtknots/UglyChristmas-Theme",
   "JOHLC/transparentblue",
   "JuanMTech/amoled_blue",
+  "JuanMTech/google-theme",
   "JuanMTech/google_dark_theme",
   "JuanMTech/google_light_theme",
```

Comment:
I guess you can remove the old dark and light themes as well as this new theme combines both?